applications/DEM_application/python_scripts/DEM_procedures.py

```diff
@@ -1311,13 +1311,19 @@ def __init__(self, post_path, name, step, which_folder):
 
 class DEMIo(object):
 
-    def __init__(self, DEM_parameters, post_path):
+    def __init__(self, DEM_parameters, post_path, all_model_parts):
```

Comment:
Changing the API is dangerous.
Is DEMIo being called from Swimming DEM?

Reply:
Some old scripts have been removed from DEM application. There are still some scripts in the swimming DEM application that GCasas will remove in the near future. 
Apart from this, the application seems to be running fine with these changes.


the complete list of benchmarks are included at the end of this message as a quick reference.

========== DEM BENCHMARKING RESULTS ==========

=========== DEM DISCONTINUUM TESTS ===========

==== TSUJI PAPER BENCHMARKS. SLIDING REGIME ==

DEM Benchmark 1: OK!........ Test 1 SUCCESSFUL (error: 0.4, time: 23.81s.)
DEM Benchmark 7: OK!........ Test 7 SUCCESSFUL (error: 0.0, time: 10.2s.)
DEM Benchmark 2: OK!........ Test 2 SUCCESSFUL (error: 0.13, time: 72.78s.)
DEM Benchmark 6: OK!........ Test 6 SUCCESSFUL (error: 3.38, time: 85.19s.)
DEM Benchmark 4: OK!........ Test 4 SUCCESSFUL (error: 4.63, time: 87.58s.)

==== THORNTON PAPER TESTS. FULL REGIME. HERTZIAN LAW ====

DEM Benchmark 11: OK!........ Test 11 (e=0.25) SUCCESSFUL
DEM Benchmark 11: OK!........ Test 11 (e=0.50) SUCCESSFUL
DEM Benchmark 11: OK!........ Test 11 (e=0.75) SUCCESSFUL
DEM Benchmark 11: OK!........ Test 11 (e=0.90) SUCCESSFUL
DEM Benchmark 8: OK!........ Test 8 SUCCESSFUL (error: 4.16, time: 87.91s.)
DEM Benchmark 3: OK!........ Test 3 SUCCESSFUL (error: 0.75, time: 122.96s.)
DEM Benchmark 5: OK!........ Test 5 SUCCESSFUL (error: 1.52, time: 123.7s.)

===== THORNTON PAPER TESTS. FULL REGIME. LINEAR LAW =====

DEM Benchmark 10: OK!........ Test 10 (e=0.25) SUCCESSFUL
DEM Benchmark 9: OK!........ Test 9 SUCCESSFUL (error: 0.97, time: 62.02s.)
DEM Benchmark 10: OK!........ Test 10 (e=0.50) SUCCESSFUL


==== WENSRICH PAPER TEST. ROLLING FRICTION ====

DEM Benchmark 12: OK!........ Test 12 SUCCESSFUL
DEM Benchmark 15: OK!........ Test 15 SUCCESSFUL


== BASIC CONTINUUM TESTS ==

DEM Benchmark 20: OK!........ Test 20 SUCCESSFUL
DEM Benchmark 21: OK!........ Test 21 SUCCESSFUL
DEM Benchmark 10: OK!........ Test 10 (e=0.75) SUCCESSFUL
DEM Benchmark 22: OK!........ Test 22 SUCCESSFUL


======== DE/FE CONTACT BENCHMARKS ==========

DEM Benchmark 13: OK!........ Test 13 SUCCESSFUL
DEM Benchmark 23: OK!........ Test 23 SUCCESSFUL
DEM Benchmark 24: OK!........ Test 24 SUCCESSFUL
DEM Benchmark 25: OK!........ Test 25 SUCCESSFUL


===== DISCONTINUUM CLUSTERS TESTS =====

DEM Benchmark 30: OK!........ Test 30 SUCCESSFUL
DEM Benchmark 31: OK!........ Test 31 SUCCESSFUL
DEM Benchmark 14: OK!........ Test 14 SUCCESSFUL
DEM Benchmark 10: OK!........ Test 10 (e=0.90) SUCCESSFUL
DEM Benchmark 16: OK!........ Test 16 SUCCESSFUL
DEM Benchmark 32: OK!........ Test 32 SUCCESSFUL
DEM Benchmark 33: OK!........ Test 33 SUCCESSFUL
DEM Benchmark 17: OK!........ Test 17 SUCCESSFUL